### PR TITLE
Adjust code for dask-jobqueue 0.7.2 compatibility

### PR DESCRIPTION
**Pull request by AdamWlodarczyk**

### CODE REVIEW

**Review by ocaisa — APPROVED** [review]
LGTM, if the tests are passing I am happy

File: jobqueue_features/clusters.py [hunk 1]
```diff
@@ -115,11 +117,42 @@ def get_features_kwarg(name, scheduler=None, queue_type=None, default=None):
     return value
 
 
+def get_base_job_kwargs():
```
Comment (ocaisa) [inline jobqueue_features/clusters.py]: Can we add a test for this?

Comment (AdamWlodarczyk) [reply jobqueue_features/clusters.py]: I will make some.

Comment (AdamWlodarczyk) [reply jobqueue_features/clusters.py]: Added tests.

**Review by ocaisa — APPROVED** [review]
LGTM Thanks @AdamWlodarczyk

**Comment by ocaisa** [comment]
tutorial test will past after merge (I think)

**Pull request merged**